nodejs/week1/src/backend/routes/reservations.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const filepath = require('path').resolve(__dirname, '..');
+
+
+function reservationsRouter() {
+
+    let reservationsData = fs.readFileSync(`${filepath}/data/reservations.json`);
+    let reservations = JSON.parse(reservationsData);
+    
+    return reservations;

```

Comment:
to simplify - replace let with const and you can directly return `JSON.parse(...` instead of adding it to a variable and then returning that variable